client/java/src/main/java/io/openlineage/client/transports/CompositeConfig.java

```diff
@@ -54,6 +63,12 @@ public CompositeConfig(
                     return nestedMap;
                   })
               .map(this::createTransportConfig)
+              .sorted(
+                  (t1, t2) -> {
+                    return Optional.ofNullable(t1.getName())
+                        .orElse("")
+                        .compareTo(Optional.ofNullable(t2.getName()).orElse(""));
+                  })
```

Comment:
is this needed? a bit cumbersome to me, wouldn't be better to make the `CompositeConfig` implement the `Comparable` interface?